server/chat-plugins/datasearch.ts

```diff
@@ -97,9 +97,9 @@ export const commands: Chat.ChatCommands = {
 		const targetGen = parseInt(cmd[cmd.length - 1]);
 		if (targetGen) target += `, mod=gen${targetGen}`;
 		const split = target.split(',').map(term => term.trim());
-		const index = split.findIndex(x => x.startsWith('maxgen'));
+		const index = split.findIndex(x => /^max ?gen/.test(x.toLowerCase()));
```

Comment:
```suggestion
		const index = split.findIndex(x => /^max\s?gen/i.test(x)));
```

Reply:
Why \s? If someone inserts a hard tab in there, is it supposed to work?

Reply:
Why shouldn't it? Should it also break if there is an nbsp?

Reply:
max&nbsp;gen #7 let's goo

(Lemme brush my teeth, will update)

Reply:
Oh right GitHub doesn't escape those